examples/rgbpp/local/launch/3-distribute-rgbpp.ts

```diff
@@ -0,0 +1,127 @@
+import { AddressPrefix, privateKeyToAddress, serializeScript } from '@nervosnetwork/ckb-sdk-utils';
+import { Collector, RgbppBtcAddressReceiver, appendCkbTxWitnesses, appendIssuerCellToBtcBatchTransfer, buildRgbppLockArgs, genBtcBatchTransferCkbVirtualTx, getXudtTypeScript, sendCkbTx, updateCkbTxWithRealBtcTxId } from '@rgbpp-sdk/ckb';
+import { sendRgbppUtxos, DataSource, ECPair, bitcoin, NetworkType, transactionToHex } from '@rgbpp-sdk/btc';
+import { BtcAssetsApi, BtcAssetsApiError } from '@rgbpp-sdk/service';
+import { RGBPP_TOKEN_INFO } from './0-rgbpp-token-info';
+
+// CKB SECP256K1 private key
+const CKB_TEST_PRIVATE_KEY = '0x0000000000000000000000000000000000000000000000000000000000000001';
+// BTC SECP256K1 private key
+const BTC_TEST_PRIVATE_KEY = '0000000000000000000000000000000000000000000000000000000000000001';
+// API docs: https://btc-assets-api.testnet.mibao.pro/docs
+const BTC_ASSETS_API_URL = 'https://btc-assets-api.testnet.mibao.pro';
+// https://btc-assets-api.testnet.mibao.pro/docs/static/index.html#/Token/post_token_generate
+const BTC_ASSETS_TOKEN = '';
+
+interface Params {
+  rgbppLockArgsList: string[];
+  receivers: RgbppBtcAddressReceiver[];
+}
+const distributeRgbppAssetOnBtc = async ({ rgbppLockArgsList, receivers }: Params) => {
+  const collector = new Collector({
+    ckbNodeUrl: 'https://testnet.ckb.dev/rpc',
+    ckbIndexerUrl: 'https://testnet.ckb.dev/indexer',
+  });
+  const isMainnet = false;
+  const ckbAddress = privateKeyToAddress(CKB_TEST_PRIVATE_KEY, {
+    prefix: isMainnet ? AddressPrefix.Mainnet : AddressPrefix.Testnet,
+  });
+  console.log('ckb address: ', ckbAddress);
+  // const fromLock = addressToScript(ckbAddress);
+
+  const network = isMainnet ? bitcoin.networks.bitcoin : bitcoin.networks.testnet;
+  const keyPair = ECPair.fromPrivateKey(Buffer.from(BTC_TEST_PRIVATE_KEY, 'hex'), { network });
+  const { address: btcAddress } = bitcoin.payments.p2wpkh({
+    pubkey: keyPair.publicKey,
+    network,
+  });
+
+  console.log('btc address: ', btcAddress);
+
+  const networkType = isMainnet ? NetworkType.MAINNET : NetworkType.TESTNET;
+  const service = BtcAssetsApi.fromToken(BTC_ASSETS_API_URL, BTC_ASSETS_TOKEN, 'https://btc-test.app');
+  const source = new DataSource(service, networkType);
+
+  const xudtType: CKBComponents.Script = {
+    ...getXudtTypeScript(isMainnet),
+    args: '0x4c1ecf2f14edae73b76ccf115ecfa40ba68ee315c96bd4fcfd771c2fb4c69e8f',
```

Comment:
suggestion: add comment explaining how to fetch the xUDT-type-script.args from the previous step.